shelley/design-spec/delegation_design_spec.tex

```diff
@@ -585,31 +585,33 @@ \subsection{Address Structure}
   \((skp, vkp)\). The address then consists of a cryptographic hash
   \(\hash{vkp}\) of the verification key \(vkp\). A witness for such an address
   contains the verification key \(vkp\) and a signature of the signing key
-  \(skp\).
+  \(skp\). We call those addresses \emph{single key addresses} or simply
+  \emph{key addresses}, and the corresponding witnesses \emph{single key
+    witnesses} or \emph{key witnesses}.
```

Comment:
Hmm, that's not the terminology we've been using in discussion so far is it? I thought we'd been using "pubkey addresses" vs "script addresses".